skema/program_analysis/TS2CAST/ts2cast.py

```diff
@@ -32,7 +33,8 @@
 class TS2CAST(object):
     def __init__(self, source_file_path: str):
         # Initialize tree-sitter
-        self.tree_sitter_fortran = Language("build/my-languages.so", "fortran")
+        tree_sitter_fortran_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "build/my-languages.so")
```

Comment:
For `build/my-languages.so`, we should create the path programmatically for OS interoperability.

Reply:
Since its wrapped in an os.path.join call, the relative path `build/my-languages.so` should be properly interpreted. However, I do like your idea of defining this somewhere as a constant. Then we could explicitly add some kind of wrapping to it to make sure its OS independent.

Reply:
Though maybe not necessarily on second look. I think `os.path.join("build", "my-languages.so")` is what we are looking for here.